<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="voltaire" errors="0" failures="55" skipped="0" tests="171" time="24.736" timestamp="2023-11-13T15:04:35.854029" hostname="fv-az1490-66"><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_priority_fee_bump]" time="0.055"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[only_max_fee_bump]" time="0.026"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_same_fee]" time="0.025"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[with_less_fee]" time="0.025"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_below_threshold]" time="0.025"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_at_threshold]" time="0.025"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_bundle_replace_op[fee_bump_above_threshold]" time="0.025"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:81: in test_bundle_replace_op
    assert new_op.send().result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-020-banned-entity-not-allowed-banned-sender]" time="0.264"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:148: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-020-banned-entity-not-allowed-banned-paymaster]" time="0.153"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:154: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-020-banned-entity-not-allowed-banned-factory]" time="0.152"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:160: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-030-throttled-entity-allowed-a-little-throttled-sender]" time="0.149"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:148: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-030-throttled-entity-allowed-a-little-throttled-paymaster]" time="0.149"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:154: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[SREP-030-throttled-entity-allowed-a-little-throttled-factory]" time="0.252"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:160: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-sender]" time="0.168"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:148: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-paymaster]" time="0.155"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:154: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_mempool_reputation_rules_all_entities[UREP-010 UREP-020-unstaked-entity-allowed-function-unstaked-factory]" time="0.182"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:160: in test_mempool_reputation_rules_all_entities
    set_reputation(
tests/utils.py:221: in set_reputation
    )
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_unstaked_sender" time="0.328" /><testcase classname="tests.single.bundle.test_bundle" name="test_max_allowed_ops_staked_sender" time="0.410" /><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_op_access_other_ops_sender_in_bundle" time="0.333"><failure message="AssertionError: assert [] == [UserOperatio...gnature='0x')]&#10;  Right contains one more item: UserOperation(sender='0x935C7eAbf869e44c74822BD7bb34568e12c64DeA', nonce='0x0', initCode='0x', callData='0x', callGasL...tionGas='0x493e0', maxFeePerGas='0xee6b2800', maxPriorityFeePerGas='0xb2d05e00', paymasterAndData='0x', signature='0x')&#10;  Full diff:&#10;    [&#10;  +  ,&#10;  -  UserOperation(sender='0x935C7eAbf869e44c74822BD7bb34568e12c64DeA',&#10;  -                nonce='0x0',&#10;  -                initCode='0x',&#10;  -                callData='0x',&#10;  -                callGasLimit='0x493e0',&#10;  -                verificationGasLimit='0xf4240',&#10;  -                preVerificationGas='0x493e0',&#10;  -                maxFeePerGas='0xee6b2800',&#10;  -                maxPriorityFeePerGas='0xb2d05e00',&#10;  -                paymasterAndData='0x',&#10;  -                signature='0x'),&#10;    ]">tests/single/bundle/test_bundle.py:319: in test_ban_user_op_access_other_ops_sender_in_bundle
    assert dump_mempool() == [user_op2]
E   AssertionError: assert [] == [UserOperatio...gnature='0x')]
E     Right contains one more item: UserOperation(sender='0x935C7eAbf869e44c74822BD7bb34568e12c64DeA', nonce='0x0', initCode='0x', callData='0x', callGasL...tionGas='0x493e0', maxFeePerGas='0xee6b2800', maxPriorityFeePerGas='0xb2d05e00', paymasterAndData='0x', signature='0x')
E     Full diff:
E       [
E     +  ,
E     -  UserOperation(sender='0x935C7eAbf869e44c74822BD7bb34568e12c64DeA',
E     -                nonce='0x0',
E     -                initCode='0x',
E     -                callData='0x',
E     -                callGasLimit='0x493e0',
E     -                verificationGasLimit='0xf4240',
E     -                preVerificationGas='0x493e0',
E     -                maxFeePerGas='0xee6b2800',
E     -                maxPriorityFeePerGas='0xb2d05e00',
E     -                paymasterAndData='0x',
E     -                signature='0x'),
E       ]</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_ban_user_sender_double_role_in_bundle" time="0.193"><failure message="Exception: expected error object, got:&#10;Ok(result='0xd990083beaa9eab619dc4e56f41f3641cefc62f7983ae7f66e59114a9c7f6d7d', id=90)">tests/utils.py:116: in assert_rpc_error
    assert response.code == code
E   AttributeError: 'Ok' object has no attribute 'code'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_bundle.py:359: in test_ban_user_sender_double_role_in_bundle
    assert_rpc_error(
tests/utils.py:119: in assert_rpc_error
    raise Exception(f"expected error object, got:\n{response}") from exc
E   Exception: expected error object, got:
E   Ok(result='0xd990083beaa9eab619dc4e56f41f3641cefc62f7983ae7f66e59114a9c7f6d7d', id=90)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_bundle" name="test_stake_check_in_bundler" time="0.043"><failure message="AttributeError: 'Error' object has no attribute 'result'">tests/single/bundle/test_bundle.py:382: in test_stake_check_in_bundler
    response = get_stake_status(paymaster_contract.address, entrypoint_contract.address)
tests/utils.py:186: in get_stake_status
    .result
E   AttributeError: 'Error' object has no attribute 'result'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_codehash" name="test_codehash_changed" time="0.458" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][paymaster][no_storage][ok]" time="0.119" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]unstaked][paymaster][storage][drop]" time="0.114" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][reference_storage][drop]" time="0.123" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][reference_storage_struct][drop]" time="0.123" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][paymaster][account_storage][ok]" time="0.149" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage][ok]" time="0.123" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][paymaster][account_reference_storage_struct][ok]" time="0.231" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop0]" time="0.225" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]unstaked][paymaster][context][drop]" time="0.103" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][paymaster][external_storage_read][drop]" time="0.122" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][out_of_gas][drop]" time="0.145"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][paymaster][sstore_out_of_gas][drop]" time="0.121" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][paymaster][no_storage][ok]" time="0.141" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][paymaster][storage][ok]" time="0.144" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][reference_storage][ok]" time="0.148" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][paymaster][reference_storage_struct][ok]" time="0.149" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][reference_storage_struct][ok]" time="0.150" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][paymaster][account_storage][ok]" time="0.146" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage][ok]" time="0.149" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][paymaster][account_reference_storage_struct][ok]" time="0.160" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]staked][paymaster][account_reference_storage_init_code][ok]" time="0.257" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[EREP-050]staked][paymaster][context][ok]" time="0.132" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_write][drop]" time="0.153" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][paymaster][external_storage_read][ok]" time="0.142"><failure message="Exception: expected result object, got:&#10;Error(code=-32502, message='paymaster : 0x8f428a0e334523d0c5e05d17e9cd2b13ab0cd0a1 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x49d2aa1cd0ba5c6da001a769d32159e7bc1519e8', data=None, id=155)">tests/utils.py:109: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/utils.py:111: in assert_ok
    raise Exception(f"expected result object, got:\n{response}") from exc
E   Exception: expected result object, got:
E   Error(code=-32502, message='paymaster : 0x8f428a0e334523d0c5e05d17e9cd2b13ab0cd0a1 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x49d2aa1cd0ba5c6da001a769d32159e7bc1519e8', data=None, id=155)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][out_of_gas][drop]" time="0.166"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][paymaster][sstore_out_of_gas][drop]" time="0.150"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][no_storage][ok]" time="0.179" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][storage][drop]" time="0.177" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][reference_storage][drop]" time="0.180" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]unstaked][factory][reference_storage_struct][drop]" time="0.286" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][factory][account_storage][ok]" time="0.185" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage][drop]" time="0.185" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][factory][account_reference_storage_struct][drop]" time="0.184" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][factory][external_storage_read][drop]" time="0.185" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-042]unstaked][factory][EXTCODEx_CALLx_undeployed_sender][ok]" time="0.181" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODESIZE_undeployed_contract][drop]" time="0.177"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODEHASH_undeployed_contract][drop]" time="0.180"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][EXTCODECOPY_undeployed_contract][drop]" time="0.179"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALL_undeployed_contract][drop]" time="0.177"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][CALLCODE_undeployed_contract][drop]" time="0.175"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][DELEGATECALL_undeployed_contract][drop]" time="0.177"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][factory][STATICCALL_undeployed_contract][drop]" time="0.178"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][out_of_gas][drop]" time="0.212"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][factory][sstore_out_of_gas][drop]" time="0.211" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][factory][no_storage][ok]" time="0.204" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-031]staked][factory][storage][ok]" time="0.205" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][reference_storage][ok]" time="0.213" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-032]staked][factory][reference_storage_struct][ok]" time="0.213" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][factory][account_storage][ok]" time="0.208" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage][ok]" time="0.211" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][factory][account_reference_storage_struct][ok]" time="0.211" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_write][drop]" time="0.209" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][factory][external_storage_read][ok]" time="0.209"><failure message="Exception: expected result object, got:&#10;Error(code=-32502, message='factory : 0xf909c4499feb5cd69b2c9476b250de3956dc4892 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x4c4da581c4b6626e9c377c1d90e320e980ab39bd', data=None, id=213)">tests/utils.py:109: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/utils.py:111: in assert_ok
    raise Exception(f"expected result object, got:\n{response}") from exc
E   Exception: expected result object, got:
E   Error(code=-32502, message='factory : 0xf909c4499feb5cd69b2c9476b250de3956dc4892 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x4c4da581c4b6626e9c377c1d90e320e980ab39bd', data=None, id=213)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][out_of_gas][drop]" time="0.335"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][factory][sstore_out_of_gas][drop]" time="0.212"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][no_storage][ok]" time="0.155" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]unstaked][account][account_storage][ok]" time="0.116" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage][ok]" time="0.121" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][drop]" time="0.198" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][account][account_reference_storage_init_code][ok]" time="0.257"><failure message="Exception: expected result object, got:&#10;Error(code=-32502, message='sender : 0xD04525500618C5cc6b717bC792B353597F0C936F insuffient stake to access 2c62b97eed54853c83bcc923c4ff7d9ae0ea77a82c24928047d554aef9826576 at contract : 0xca22776afd692693cc8779ae77a5f0a0462e6549', data=None, id=227)">tests/utils.py:109: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/utils.py:111: in assert_ok
    raise Exception(f"expected result object, got:\n{response}") from exc
E   Exception: expected result object, got:
E   Error(code=-32502, message='sender : 0xD04525500618C5cc6b717bC792B353597F0C936F insuffient stake to access 2c62b97eed54853c83bcc923c4ff7d9ae0ea77a82c24928047d554aef9826576 at contract : 0xca22776afd692693cc8779ae77a5f0a0462e6549', data=None, id=227)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-022]unstaked][paymaster][account_reference_storage_init_code][drop1]" time="0.228" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]unstaked][account][account_reference_storage_struct][ok]" time="0.121" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]unstaked][account][external_storage_read][drop]" time="0.115" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][out_of_gas][drop]" time="0.141"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]unstaked][account][sstore_out_of_gas][drop]" time="0.118"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-000]staked][account][no_storage][ok]" time="0.136" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-010]staked][account][account_storage][ok]" time="0.137" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage][ok]" time="0.143" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-021]staked][account][account_reference_storage_struct][ok]" time="0.141" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][out_of_gas][drop]" time="0.160"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-020]staked][account][sstore_out_of_gas][drop]" time="0.172"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_write][drop]" time="0.139" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[STO-033]staked][account][external_storage_read][ok]" time="0.138"><failure message="Exception: expected result object, got:&#10;Error(code=-32502, message='sender : 0xEDA998481992f13fb0244c98b9837b873Af33804 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x73c970d445df5b44ac56f8ed797b20afae89bb28', data=None, id=253)">tests/utils.py:109: in assert_ok
    assert response.result
E   AttributeError: 'Error' object has no attribute 'result'

The above exception was the direct cause of the following exception:
tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/utils.py:111: in assert_ok
    raise Exception(f"expected result object, got:\n{response}") from exc
E   Exception: expected result object, got:
E   Error(code=-32502, message='sender : 0xEDA998481992f13fb0244c98b9837b873Af33804 banned access to slot aed3469875ad6750e818d667b5bd08036a9d5a353af5c0e81a03431a837a7b39 at contract : 0x73c970d445df5b44ac56f8ed797b20afae89bb28', data=None, id=253)</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-011]unstaked][account][entryPoint_call_balanceOf][drop]" time="0.123"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-061]unstaked][account][eth_value_transfer_forbidden][drop]" time="0.117"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-053]unstaked][account][eth_value_transfer_entryPoint][ok]" time="0.129" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-052]unstaked][account][eth_value_transfer_entryPoint_depositTo][ok]" time="0.127" /><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODESIZE_undeployed_contract][drop]" time="0.109"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODEHASH_undeployed_contract][drop]" time="0.108"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][EXTCODECOPY_undeployed_contract][drop]" time="0.112"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODESIZE_entrypoint][drop]" time="0.113"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODEHASH_entrypoint][drop]" time="0.116"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-054]unstaked][account][EXTCODECOPY_entrypoint][drop]" time="0.112"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALL_undeployed_contract][drop]" time="0.128"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][CALLCODE_undeployed_contract][drop]" time="0.111"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][DELEGATECALL_undeployed_contract][drop]" time="0.108"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-041]unstaked][account][STATICCALL_undeployed_contract][drop]" time="0.111"><failure message="AttributeError: 'Ok' object has no attribute 'message'">tests/single/bundle/test_storage_rules.py:769: in test_rule
    case.assert_func(response)
tests/single/bundle/test_storage_rules.py:19: in assert_error
    assert_rpc_error(response, response.message, RPCErrorCode.BANNED_OPCODE)
E   AttributeError: 'Ok' object has no attribute 'message'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.bundle.test_storage_rules" name="test_rule[[OP-062]unstaked][account][CALL_undeployed_contract_allowed_precompile][ok]" time="0.117" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GAS]" time="0.079" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[NUMBER]" time="0.082" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[TIMESTAMP]" time="0.079" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[COINBASE]" time="0.085" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[DIFFICULTY]" time="0.080" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BASEFEE]" time="0.085" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASLIMIT]" time="0.080" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[GASPRICE]" time="0.082" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFBALANCE]" time="0.084" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BALANCE]" time="0.082" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[ORIGIN]" time="0.085" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[BLOCKHASH]" time="0.084" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE]" time="0.085" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[CREATE2]" time="0.087" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_banned_opcode[SELFDESTRUCT]" time="0.086" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS CALL]" time="0.127" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_account_allowed_opcode_sequence[GAS DELEGATECALL]" time="0.095" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GAS]" time="0.103" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[NUMBER]" time="0.099" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[TIMESTAMP]" time="0.099" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[COINBASE]" time="0.105" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[DIFFICULTY]" time="0.099" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BASEFEE]" time="0.104" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASLIMIT]" time="0.102" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[GASPRICE]" time="0.105" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFBALANCE]" time="0.103" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BALANCE]" time="0.105" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[ORIGIN]" time="0.104" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[BLOCKHASH]" time="0.109" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE]" time="0.107" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[CREATE2]" time="0.108" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_paymaster_banned_opcode[SELFDESTRUCT]" time="0.110" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GAS]" time="0.161" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[NUMBER]" time="0.163" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[TIMESTAMP]" time="0.164" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[COINBASE]" time="0.165" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[DIFFICULTY]" time="0.164" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BASEFEE]" time="0.166" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASLIMIT]" time="0.166" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[GASPRICE]" time="0.166" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFBALANCE]" time="0.164" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BALANCE]" time="0.166" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[ORIGIN]" time="0.165" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[BLOCKHASH]" time="0.168" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE]" time="0.170" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[CREATE2]" time="0.172" /><testcase classname="tests.single.opbanning.test_op_banning" name="test_factory_banned_opcode[SELFDESTRUCT]" time="0.172" /><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[with_factory]" time="0.126"><failure message="AttributeError: 'str' object has no attribute 'address'">tests/single/reputation/test_reputation.py:71: in test_staked_entity_reputation_threshold
    ops_included = next(
tests/single/reputation/test_reputation.py:72: in &lt;genexpr&gt;
    (rep for rep in reputations if rep.address == paymaster_contract.address), {}
E   AttributeError: 'str' object has no attribute 'address'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.reputation.test_reputation" name="test_staked_entity_reputation_threshold[without_factory]" time="0.067"><failure message="AttributeError: 'str' object has no attribute 'address'">tests/single/reputation/test_reputation.py:71: in test_staked_entity_reputation_threshold
    ops_included = next(
tests/single/reputation/test_reputation.py:72: in &lt;genexpr&gt;
    (rep for rep in reputations if rep.address == paymaster_contract.address), {}
E   AttributeError: 'str' object has no attribute 'address'</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_chainId" name="test_eth_chainId[]" time="0.008" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas[]" time="0.029"><failure message="jsonschema.exceptions.ValidationError: 'verificationGasLimit' is a required property&#10;&#10;Failed validating 'required' in schema:&#10;    {'properties': {'callGasLimit': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',&#10;                                                'type': 'string'},&#10;                                               {'type': 'number'}],&#10;                                     'title': 'callGasLimit'},&#10;                    'preVerificationGas': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',&#10;                                                      'type': 'string'},&#10;                                                     {'type': 'number'}],&#10;                                           'title': 'preVerificationGas'},&#10;                    'verificationGasLimit': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',&#10;                                                        'type': 'string'},&#10;                                                       {'type': 'number'}],&#10;                                             'title': 'verificationGasLimit'}},&#10;     'required': ['callGasLimit',&#10;                  'preVerificationGas',&#10;                  'verificationGasLimit'],&#10;     'title': 'UserOperation Gas parameters',&#10;     'type': 'object'}&#10;&#10;On instance:&#10;    {'callGasLimit': '0xab10',&#10;     'deadline': 281474976710655,&#10;     'preVerificationGas': 48220,&#10;     'verificationGas': 334329}">tests/single/rpc/test_eth_estimateUserOperationGas.py:20: in test_eth_estimateUserOperationGas
    validate(instance=response.result, schema=schema)
.venv/lib/python3.10/site-packages/jsonschema/validators.py:1121: in validate
    raise error
E   jsonschema.exceptions.ValidationError: 'verificationGasLimit' is a required property
E   
E   Failed validating 'required' in schema:
E       {'properties': {'callGasLimit': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',
E                                                   'type': 'string'},
E                                                  {'type': 'number'}],
E                                        'title': 'callGasLimit'},
E                       'preVerificationGas': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',
E                                                         'type': 'string'},
E                                                        {'type': 'number'}],
E                                              'title': 'preVerificationGas'},
E                       'verificationGasLimit': {'oneOf': [{'pattern': '^0x([1-9a-f]+[0-9a-f]*|0)$',
E                                                           'type': 'string'},
E                                                          {'type': 'number'}],
E                                                'title': 'verificationGasLimit'}},
E        'required': ['callGasLimit',
E                     'preVerificationGas',
E                     'verificationGasLimit'],
E        'title': 'UserOperation Gas parameters',
E        'type': 'object'}
E   
E   On instance:
E       {'callGasLimit': '0xab10',
E        'deadline': 281474976710655,
E        'preVerificationGas': 48220,
E        'verificationGas': 334329}</failure><system-out>--------------------------------- Captured Log ---------------------------------

--------------------------------- Captured Out ---------------------------------

</system-out><system-err>--------------------------------- Captured Err ---------------------------------

</system-err></testcase><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_execution_revert" time="0.027" /><testcase classname="tests.single.rpc.test_eth_estimateUserOperationGas" name="test_eth_estimateUserOperationGas_simulation_revert" time="0.022" /><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash[]" time="0.143" /><testcase classname="tests.single.rpc.test_eth_getUserOperationByHash" name="test_eth_getUserOperationByHash_error" time="0.003" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt[]" time="0.153" /><testcase classname="tests.single.rpc.test_eth_getUserOperationReceipt" name="test_eth_getUserOperationReceipt_error" time="0.002" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation[]" time="0.126" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_revert" time="0.034" /><testcase classname="tests.single.rpc.test_eth_sendUserOperation" name="test_eth_sendUserOperation_invalid_signature" time="0.024" /><testcase classname="tests.single.rpc.test_eth_supportedEntryPoints" name="test_eth_supportedEntryPoints[]" time="0.012" /></testsuite></testsuites>